bitnami/fluentd/values-production.yaml

```diff
@@ -13,7 +13,7 @@ global: {}
 image:
   registry: docker.io
   repository: bitnami/fluentd
-  tag: 1.11.2-debian-10-r18
+  tag: 1.11.2-debian-10-r19
```

Comment:
Could you explain the reason for bumping this version?

Reply:
My apologies - I originally had a comment in the PR request to explain this when it was draft, but removed it. Oops - sorry.

To work correctly for the **forwarder** running as `root` with no special Linux capabilities, this change is dependent on a couple of changes to the start scripts merged in https://github.com/bitnami/bitnami-docker-fluentd/pull/14 - which is only released onto Docker Hub in `-r19` of the Docker image. I was waiting for the `bitnami-bot` automation to update this version in the chart automatically, but that doesn't seem to have happened yet. I'm not sure what cycle/process these bot-tag-updates are made on.

Am I best to wait for that automation to flow through and make this change for me, and we merge after that?

Reply:
Ok, thanks for letting me know!